pod1: {
    x_loc: 35.55,
    y_loc: 48.12,
    theta: 1.57
}